metrics/fastly/process-raw-nix-logs.nix

```diff
@@ -0,0 +1,25 @@
+{ config, pkgs, ... }: {
+
+  users.users.nix-metrics =
+    { isNormalUser = true;
+      description = "Nix Metrics Collection";
+    };
```

Comment:
Do we need a statically allocated user (and uid), or can the service use the `DynamicUser` feature, which creates an ad hoc user, when the service runs?

Reply:
That's probably possible. But it does need to have the AWS credentials somewhere.

Reply:
In which case we could probably use `LoadCredential`.

```
   systemd.services.process-raw-nix-logs.serviceConfig.LoadCredential = [
     "aws-cred:/path/to/actual/aws-cred-file"
   ];
```
The secret will then be provided at `$CREDENTIALS_DIRECTORY/aws-cred`.

Reply:
Maybe you can share the secret with me via Bitwarden and I'll create the age file? Feel free to reach out on Matrix to expedite this process.